examples/ilu-preconditioned-solver/doc/results.dox

```diff
@@ -24,10 +25,10 @@ Solution (x):
 0.0107016
 0.0121141
 0.0123025
-Residual norm sqrt(r^T r): 
+Residual norm sqrt(r^T r):
 %%MatrixMarket matrix array real general
 1 1
-9.08137e-16
+1.46249e-08
```

Comment:
the original one is from `reduction_factor = 1e-15`. current `reduction_factor = 1e-7`